sm-core-model/src/main/java/com/salesmanager/core/model/catalog/product/image/ProductImage.java

```diff
@@ -29,7 +29,7 @@
 	
 	@Id
 	@Column(name = "PRODUCT_IMAGE_ID")
-	@TableGenerator(name = "TABLE_GEN", table = "SM_SEQUENCER", pkColumnName = "SEQ_NAME", valueColumnName = "SEQ_COUNT", pkColumnValue = "PRODUCT_IMG_SEQ_NEXT_VAL")
+	@TableGenerator(name = "TABLE_GEN", table = "SM_SEQUENCER", pkColumnName = "SEQ_NAME", valueColumnName = "SEQ_COUNT", pkColumnValue = "CATEGORY_IMG_SEQ_NEXT_VAL")
```

Comment:
I think this should be reverted, right?
As per my understanding, this is ProductImage and no changes needed here.

Reply:
yes, you are right.